### PR TITLE
Ataymano/no train fix

### PR DESCRIPTION
**Pull request by ataymano**
Offline experimentation fix

### CODE REVIEW

**Comment by JohnLangford** [comment]
Appveyor build is failing.

File: vowpalwabbit/example.cc [hunk 1]
```diff
@@ -241,4 +242,11 @@ void clear_seq_and_finish_examples(vw& all, multi_ex& ec_seq)
   ec_seq.clear();
 }
 
+void return_unused_examples(vw& all, v_array<example*>& examples)
```
Comment (JohnLangford) [inline vowpalwabbit/example.cc]: return_multiline_example instead?

File: vowpalwabbit/parser.cc [hunk 2]
```diff
@@ -787,6 +787,19 @@ example& get_unused_example(vw* all)
   }
 }
 
+void return_unused_example(vw& all, example& ec)
```
Comment (JohnLangford) [inline vowpalwabbit/parser.cc]: Why can't we use finish_example() instead?

**Comment by ataymano** [comment]
>           Appveyor build is failing.
It was cancelled. Now it passed

**Comment by JohnLangford** [comment]
Why did the build fail?

**Pull request merged**